WORKSPACE.in

```diff
@@ -142,6 +147,11 @@ new_http_archive(
     strip_prefix = "yaml-cpp-yaml-cpp-0.5.3",
     url = "file:///home/tmp/yaml-cpp-0.5.3.zip",
 )
+#new_local_repository(
```

Comment:
please verify ./apollo.sh build in x86_64 platform too.

Reply:
I have verified on x86_64 and ARM64 platform with these commands:
./apollo.sh build
./apollo.sh build_gpu
./apollo.sh build_opt_gpu
All Build passed!